src/styles/page.scss

```diff
@@ -0,0 +1,10 @@
+* {
+  margin: 0;
+  transition: all 300ms;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document. Same for transition. Avoid using anything except box-sizing to *

Reply:
and list all transition properties instead of using 'all'. It's also important for the performance